build/stage-release-appcenter.yml

```diff
@@ -10,56 +9,6 @@ parameters:
   appCenterDistributionGroup: ''
 
 jobs:
-- deployment: AppCenter_UWP
```

Comment:
Isn't that already done in the dotnet-6 branch?

Reply:
Probably, I'm not aware, I'm doing to this to remove the failed build notification on main branch builds

Reply:
I don't think the problem would be UWP only, I know that iOS production build fail every time because of certificate issue I think.

`No iOS signing identities match the specified provisioning profile.`